Mashael_Almosallam/week_04/day_03/bank.rb

```diff
@@ -0,0 +1,67 @@
+class Bank
+  attr_accessor :name, :location, :accounts
+
+  def initialize(name, location, accounts)
+    @name = name
+    @location = location
+    @accounts = accounts
```

Comment:
I don't think this is doing what you expect it. We can talk about this further if you'd like.